DESIGN-PROPOSALS.md

```diff
@@ -0,0 +1,93 @@
+
+**Checklist**:
+
+- [ ] Prerequisite 1
+- [ ] Prerequisite 2
+- [ ] Component 1
+- [ ] Component 2
+- [ ] Docs
+- [ ] Tests
+
+
+
+## Summary/Abstract
+
+Provide a high-level summary of the proposed change. Keep it short.
+
+This design process is usually around adding a series of standards around recording change notes for the project. The intent is to make the process of building the release changelog faster and easier and track decisions for the future. It is usually a combination of a feature and effort-tracking document, a product requirements document, and a design document.
+
+## Background/Motivation
+
+Provide the motivation for the feature and any context required to understand the motivation. The motivation should justify any potential impact and why it is important. Often these proposals start as an issue, forum post, email and it's helpful to link to it in this section to provide context and credit the right people for the idea.
+
+It is vital for projects to be able to track the chain of custody for a proposed enhancement from conception through implementation which can sometimes be difficult to do in a single Github issue, especially when it is a larger design decision or cuts across multiple areas of the project.
+
+The purpose of the design proposal processes is to reduce the amount of "siloed knowledge" in a community. By moving decisions from a smattering of mailing lists, video calls, slack messages, Github exchanges, and hallway conversations into a well tracked artifact, the process aims to enhance communication and discoverability.
+
+## (Optional) User/User Story
+
+Detail the things that people will be able to do with the feature if it is implemented. Include as much detail as possible so that people can understand the "how" of the system. This can also be combined with the background/motivation above.
+
+## Goals
+
+The desired goal and list goals that the design achieves. What is it trying to achieve? How will you know that it has succeeded?
+
+## Non-Goals
+
+Describe what is out of scope for the design proposal. Listing non-goals helps to focus discussion and make progress.
+
+## Proposal
+
+This is where we get down to the specifics of what the proposal actually is. It should have enough detail that reviewers can understand exactly what you're proposing, but should not include things like API designs or implementation. Describe the desired outcome and how to measure success.
+
+## Design Details
+
+This section should contain enough information that the specifics of the change are understandable. This may include API specs (though not always required) or even code snippets. If there's any ambiguity about HOW your proposal will be implemented, this is the place to discuss them. This can also be combined with the proposal section above. It should also address how the solution is backward compatible and how to deal with these incompatibilities, possibly with defaulting or migrations.
```

Comment:
```suggestion
This section should contain enough information that the specifics of the change are understandable. This may include API specs (though not always required) or even code snippets. If there's any ambiguity about HOW your proposal will be implemented, this is the place to discuss them. This can also be combined with the proposal section above. It should also address how the solution is backward compatible and how to deal with these incompatibilities, possibly with defaulting or migrations. It may be useful to refer back to the goals and non-goals to assist in articulating the "why" behind your approach.
```

Reply:
I'd change "specifics" to something more, umm, specific. Like, should the section contain enough informaiton that the "proposed implementation" can be understood? Is this still for more depth to the product direction and not tech/implementation details?